.github/workflows/ci.yml

```diff
@@ -85,6 +86,8 @@ jobs:
           S3_SECRET_KEY: ${{ secrets.s3_secret_key }}
           S3_ENDPOINT: https://s3.sbg.cloud.ovh.net/
           S3_REGION: sbg
+          S3_BUCKET_NAME: quetz
+          QUETZ_S3_BUCKET_NAME: quetz
```

Comment:
Can you brielfy elaborate what's the purpose of this env var?

Reply:
As I found out with https://github.com/fsspec/s3fs/issues/824, we need to specify the bucket_name in the paths for s3fs, we need to know how the bucket is called that we write to. Previously (at least for aws s3) this information could be encoded in `S3_ENDPOINT`, however, f3fs doesn't work with that (bucket cannot be encoded in URL). Therefore we need another variable explicitly stating the bucket_name.

Reply:
But why do we need `S3_BUCKET_NAME` and `QUETZ_S3_BUCKET_NAME`?

Reply:
I already removed `QUETZ_S3_BUCKET_NAME` again, I just tested the `QUETZ_` prefix to make sure that it wouldn't be mapped to `S3_BUCKET_NAME` in the tests (as the env variable was missing)